pkg/apiserver/slowquery/queries.go

```diff
@@ -14,93 +14,16 @@
 package slowquery
 
 import (
-	"fmt"
-	"reflect"
 	"strings"
 
 	"github.com/jinzhu/gorm"
-	"github.com/thoas/go-funk"
 )
 
 const (
-	SlowQueryTable = "INFORMATION_SCHEMA.CLUSTER_SLOW_QUERY"
-	SelectStmt     = "*, (UNIX_TIMESTAMP(Time) + 0E0) AS timestamp"
+	slowQueryTable = "INFORMATION_SCHEMA.CLUSTER_SLOW_QUERY"
+	selectStmt     = "*, (UNIX_TIMESTAMP(Time) + 0E0) AS timestamp"
```

Comment:
let's drop this constant and inline it in the code instead. I believe it brings more confusion than simply hardcoding, since this constant can not be generally used in other places.